### DISCUSSION TITLE
Synology: docker or not, and HTTPS problem.

### DISCUSSION THREAD

**Discussion by willio** [dani-garcia/vaultwarden#1906] [Q&A]
My quick and flawless installation via docker was up and runnning, up until I tried to register.

I use the Application Portal reverse proxy to redirect (HTTPS) sub.mydomain.com:port, to (HTTP) localhost:80 where my vaultwarden install is. However, accessing https://sub.mydomain.com:port returned a connection error (can't establish secure connection), but http://sub.mydomain.com:port renders the page fine, up until I tried to create an account, where HTTPS is required.

I have sub.mydomain.com configured correctly with an LE certificate, and made sure that the vaultwarden docker instance uses that certificate.

Any other synology users out there having this problem? Any idea how to solve this? Thanks.

**Comment by BobWs** [comment]
If you are using Synology RP than you don't need to add the port in the URL just your (sub).domain.com
e.g. https://vault.yourdomain.com that should render your page correctly if you have configured RP correctly.

Maybe you could post some screenshot? And a bit more info how you configured RP and Bitwarden?

**Reply by willio** [reply]
Found the underlying problem, it's a NAT port issue.
